CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
```

Comment:
Doesn't this trigger errors on Ubuntu 16.04?

Reply:
Merging this, but we can change this as needed.

Reply:
it doesn't. tested on my system (15.04 with default Cmake 3.5.2).